moveit_ros/planning/planning_request_adapter_plugins/src/fix_start_state_collision.cpp

```diff
@@ -71,19 +72,30 @@ class FixStartStateCollision : public planning_request_adapter::PlanningRequestA
     else
       ROS_INFO_STREAM("Param '" << JIGGLE_PARAM_NAME << "' was set to " << jiggle_fraction_);
 
-    if (!nh.getParam(ATTEMPTS_PARAM_NAME, sampling_attempts_))
+    if (!nh.getParam(DISTANCE_PARAM_NAME, max_jiggle_distance_))
     {
-      sampling_attempts_ = 100;
-      ROS_INFO_STREAM("Param '" << ATTEMPTS_PARAM_NAME << "' was not set. Using default value: " << sampling_attempts_);
+      max_jiggle_distance_ = 0.02;
+      ROS_INFO_STREAM("Param '" << DISTANCE_PARAM_NAME
+                                << "' was not set. Using default value: " << max_jiggle_distance_);
+    }
+    else
```

Comment:
I'm not sure if that is part of the coding style but I prefer always placing brackets even for a single term after `if`/`else`